### PR TITLE
[AGW Documentations] AGW Static IP set up

### PR DESCRIPTION
**Pull request by jaredmullane**
## Summary

Adding instructions for partners who were looking to do non-DHCP AGW installs. Related to T79129909. 

## Test Plan

docs/docusaurus/create_docusaurus_website.sh

### CODE REVIEW

File: docs/docusaurus/versioned_docs/version-1.0.1/lte/setup_deb.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ satisfies the following requirements:
   enp2s0 with your current interfaces name in this guideline.
   One port is for the SGi interface (default: enp1s0) and one for the S1
   interface (default: enp2s0). Note that the `agw_install.sh` script will
-  rename the `enp1s0` interface to `eth0`.
+  rename the `enp1s0` interface to `eth0`. By default, `agw_install.sh` will configure the `eth0` interface for DHCP. In order to make any custom changes, you will need to alter the `agw_install.sh` script.
```
Comment (pshelar) [inline docs/docusaurus/versioned_docs/version-1.0.1/lte/setup_deb.md:21]: @jaredmullane there is orc8r API and NMS option to set static IP for SGi interface, can you document that

**Review by pshelar — CHANGES_REQUESTED** [review]

**Comment by ekowtaylor** [comment]
@jaredmullane are you moving forward with this?

**Comment by ulaskozat** [comment]
What is the status here?

**Comment by jaredmullane** [comment]
Going to close this as we have updated the steps in the FAQ.

**Pull request closed without merging**